hail/src/test/scala/is/hail/expr/ir/IRSuite.scala

```diff
@@ -1027,8 +1027,8 @@ class IRSuite extends SparkSuite {
     implicit val execStrats = Set(ExecStrategy.CxxCompile)
```

Comment:
Probably don't need this line anymore?

Reply:
Although there's not too much harm in keeping it in if we change this to later be able to execute multiple paths.